packages/core/src/behavior.ts

```diff
@@ -0,0 +1,59 @@
+import { ActorRef, Behavior, EventObject, Observer } from '.';
+import { toActorRef } from './Actor';
+import { toEventObject, toObserver } from './utils';
+
+/**
+ * Returns an actor behavior from a reducer and its initial state.
+ *
+ * @param transition The pure reducer that returns the next state given the current state and event.
+ * @param initialState The initial state of the reducer.
+ * @returns An actor behavior
+ */
+export function fromReducer<TState, TEvent extends EventObject>(
+  transition: (state: TState, event: TEvent) => TState,
+  initialState: TState
+): Behavior<TEvent, TState> {
+  return {
+    transition,
+    initialState
+  };
+}
+
+interface SpawnBehaviorOptions {
+  id?: string;
+  parent?: ActorRef<any>;
+}
+
+export function spawnBehavior<TActorEvent extends EventObject, TEmitted>(
```

Comment:
nit pick: I would rename `TActorEvent` to just `TEvent`. 

Right now the signature of this function is using `SpawnBehaviorOptions` and `TActorEvent` - so it IMHO is a little bit confusing if the main thing here is a behavior or an actor. Making this generic less specific~ would avoid this mismatch in naming entirely.